src/ws/session.c

```diff
@@ -737,9 +738,16 @@ perform_gssapi (const char *rhost,
   char *str = NULL;
   OM_uint32 caps = 0;
   int res;
+  struct stat sb;
 
   res = PAM_AUTH_ERR;
 
+  if (!getenv ("COCKPIT_KEEP_KTAB") &&
+      stat (DEFAULT_KTAB, &sb) == 0)
```

Comment:
Nitpick: `access(F_OK)` avoids having to introduce the `sb`, as it's not actually used.

AFAIUI `$COCKPIT_KEEP_KTAB` is only being used to short-circuit this path for the unit tests, right? (The variable name is a bit misleading -- perhaps `COCKPIT_TEST_KEEP_KTAB`, like with `COCKPIT_TEST_KEEP_PATH`?)

Reply:
changed